app/mailers/feedback_mailer.rb

```diff
@@ -13,6 +13,6 @@ def feedback(sending_user, form_values)
   end
 
   def self.support_email
-    ENV['SUPPORT_EMAIL'] || 'gatewaycommunicator@gsa.gov' # not sensitive, so hard coding
+    ENV['SUPPORT_EMAIL'] || 'tickets@18f.uservoice.com' # not sensitive, so hard coding
```

Comment:
Cool, that works for now. Maybe `help.c2.18f.gov` eventually if we need a different subdomain.